Arrowgene.Ddon.GameServer/Characters/CraftManager.cs

```diff
@@ -171,22 +168,25 @@ public static CraftCalculationResult CalculateEquipmentQuality(Item refineMateri
                     case 8036 or 8068:
                         RandomQuality = 2;
                         greatSuccessValue = 3;
+                        exp = CalculateQualityExp(itemRank, false);
                         break;
                     // WhiteDragon Rocks (Tier3)
                     case 8052 or 8084:
                         RandomQuality = 2;
                         greatSuccessValue = 3;
-                        greatSuccessOdds = 5;
+                        greatSuccessOdds = 25;
+                        exp = CalculateQualityExp(itemRank, true);
                         break;
                     // Standard Rocks (Tier1)
                     case 8035 or 8067:
                         RandomQuality = 1;
                         greatSuccessValue = 2;
+                        exp = 2;
```

Comment:
Should this exp value be returned by `CalculateQualityExp` as well incase there ends up being other modifiers?